src/main/kotlin/uk/gov/dwp/dataworks/services/ConfigurationResolver.kt

```diff
@@ -57,10 +57,12 @@ class ConfigurationResolver {
 enum class ConfigKey(val key: String, val isList: Boolean) {
     AWS_REGION("orchestrationService.aws_region", false),
     COGNITO_USER_POOL_ID("orchestrationService.cognito_user_pool_id", false),
-    LOAD_BALANCER_NAME("orchestrationService.load_balancer_name",false),
-    LOAD_BALANCER_PORT("orchestrationService.load_balancer_port",false),
-    USER_CONTAINER_TASK_DEFINITION("orchestrationService.user_container_task_definition",false),
+    LOAD_BALANCER_NAME("orchestrationService.load_balancer_name", false),
+    LOAD_BALANCER_PORT("orchestrationService.load_balancer_port", false),
     ECS_CLUSTER_NAME("orchestrationService.ecs_cluster_name", false),
     USER_CONTAINER_URL("orchestrationService.user_container_url", false),
-    USER_CONTAINER_PORT("orchestrationService.user_container_port", false)
+    USER_CONTAINER_PORT("orchestrationService.user_container_port", false),
+    USER_TASK_EXECUTION_ROLE_ARN("orchestrationService.user_task_execution_role_arn", false),
+    USER_TASK_ROLE_ARN("orchestrationService.user_task_role_arn", false),
+    ECR_ENDPOINT("orchestrationService.ecr_endpoint", false)
```

Comment:
Do we need to add these to the TF code as well?

Reply:
They're created by terraform and needed as part of the Task Definition creation ([here](https://github.com/dwp/orchestration-service/pull/49/files#diff-8c6199d4d9242737e9fc6c2596ea8178R72)), if you have a suggestion on how to do this differently I'm happy to change it

Reply:
No, I was just wondering where the TF code was, you've added it now do all good!